File: Python03_Android/AndroidStudioProjects/EditBoxTest/app/src/main/java/com/example/editboxtest/MainActivity.java
package com.example.editboxtest;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {


    private TextView textView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        textView = (TextView)findViewById(R.id.textView);
        Button button = findViewById(R.id.button);
        EditText editText = findViewById(R.id.editText);

        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                String str = editText.getText().toString();
                textView.setText(str);
            }
        });
    }
}

File: JAVA01_AndroidStudio/001.basic/app/src/main/java/com/example/androidapp01/MainActivity.kt
package com.example.androidapp01

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var txtHello : TextView = findViewById(R.id.txtHello)
        var btnButton : Button = findViewById(R.id.btnOk)
        btnButton.setOnClickListener(){
            txtHello.setText("Hello Android App!!")
        }
    }
}
